src/metalsmith/_redirects

```diff
@@ -75,3 +75,59 @@ https://concepts.effectivealtruism.org/concepts/unknown-existential-risks/ https
+https://concepts.effectivealtruism.org/counterfactual-considerations https://forum.effectivealtruism.org/tag/counterfactual-reasoning 301!
+https://concepts.effectivealtruism.org/replaceability https://forum.effectivealtruism.org/tag/replaceability 301!
+https://concepts.effectivealtruism.org/the-importance-of-crucial-considerations https://forum.effectivealtruism.org/tag/crucial-consideration 301!
+https://concepts.effectivealtruism.org/thinking-at-the-margin https://forum.effectivealtruism.org/tag/marginal-impact 301!
+https://concepts.effectivealtruism.org/thinking-globally https://forum.effectivealtruism.org/tag/cosmopolitanism 301!
+https://concepts.effectivealtruism.org/measuring-healthy-life-years https://forum.effectivealtruism.org/tag/health-adjusted-life-years 301!
+https://concepts.effectivealtruism.org/value-of-information https://forum.effectivealtruism.org/tag/value-of-information 301!
+https://concepts.effectivealtruism.org/discounting-the-future https://forum.effectivealtruism.org/tag/temporal-discounting 301!
+https://concepts.effectivealtruism.org/assessing-and-assimilating-evidence https://forum.effectivealtruism.org/tag/assessing-and-assimilating-evidence 301!
+https://concepts.effectivealtruism.org/cognitive-biases https://forum.effectivealtruism.org/tag/cognitive-bias 301!
+https://concepts.effectivealtruism.org/randomized-controlled-trials https://forum.effectivealtruism.org/tag/randomized-controlled-trials 301!
+https://concepts.effectivealtruism.org/using-models https://forum.effectivealtruism.org/tag/models 301!
+https://concepts.effectivealtruism.org/uncertainty-within-models https://forum.effectivealtruism.org/tag/model-uncertainty 301!
+https://concepts.effectivealtruism.org/moral-cooperation https://forum.effectivealtruism.org/tag/moral-cooperation 301!
+https://concepts.effectivealtruism.org/moral-trade https://forum.effectivealtruism.org/tag/moral-trade 301!
+https://concepts.effectivealtruism.org/game-theory https://forum.effectivealtruism.org/tag/game-theory 301!
+https://concepts.effectivealtruism.org/unilateralists-curse https://forum.effectivealtruism.org/tag/unilateralist-s-curse 301!
+https://concepts.effectivealtruism.org/indirect-effects https://forum.effectivealtruism.org/tag/indirect-long-term-effects 301!
+https://concepts.effectivealtruism.org/information-hazards https://forum.effectivealtruism.org/tag/information-hazard 301!
+https://concepts.effectivealtruism.org/long-term-indirect-effects https://forum.effectivealtruism.org/tag/indirect-long-term-effects 301!
+https://concepts.effectivealtruism.org/naive-consequentialism-vs-sophisticated-consequentialism https://forum.effectivealtruism.org/tag/naive-vs-sophisticated-consequentialism 301!
+https://concepts.effectivealtruism.org/inside-and-outside-view https://forum.effectivealtruism.org/tag/inside-vs-outside-view 301!
```

Comment:
[Nit] how did you manage to not have a newline at the end of the file in vscode?